cfn-resources/serverless-instance/cmd/resource/resource.go

```diff
@@ -307,59 +310,58 @@ func readServerlessInstance(cluster *mongodbatlas.Cluster, profile *string) (ser
 
 	if cluster.ConnectionStrings != nil {
 		serverless.ConnectionStrings = &ServerlessInstanceConnectionStrings{
-			StandardSrv:     &cluster.ConnectionStrings.StandardSrv,
+			StandardSrv:     cluster.ConnectionStrings.StandardSrv,
 			PrivateEndpoint: readPrivateEndpoint(cluster.ConnectionStrings.PrivateEndpoint),
 		}
 	}
-	serverless.CreateDate = &cluster.CreateDate
-	serverless.MongoDBVersion = &cluster.MongoDBVersion
+	serverless.CreateDate = admin.PtrString(cluster.CreateDate.String())
```

Comment:
what is PtrString used for? what are serverless.CreateDate and cluster.CreateDate types?

Reply:
it returns a `*string` given the returned value from `cluster.CreateDate.String() `

Reply:
what about this so you also make sure the date format is correct?   serverless.CreateDate = util.TimePtrToStringPtr(cluster.CreateDate)

Reply:
we don't want to use String() function in time.Time, it's not ISO,   in my current implementation it is:
s := t.Format("2006-01-02 15:04:05.999999999 -0700 MST")

Reply:
good point. I updated it to use `util.TimePtrToStringPtr` which set the format to `RFC3339`